rpc/context.go

```diff
@@ -60,26 +60,14 @@ func ContextPeerConnection(ctx context.Context) (*webrtc.PeerConnection, bool) {
 	return pc.(*webrtc.PeerConnection), true
 }
 
-// contextWithAuthMetadata attaches authentication metadata to the given context.
-func contextWithAuthMetadata(ctx context.Context, authMD map[string]string) context.Context {
-	return context.WithValue(ctx, ctxKeyAuthMetadata, authMD)
-}
-
-// ContextAuthMetadata returns authentication metadata. It may be nil if the value was never set.
-func ContextAuthMetadata(ctx context.Context) map[string]string {
```

Comment:
refactored this into claims. also ultimately decided not to pass claims to handlers because it's harder to signal to the handlers if the claims are confidential or not

Reply:
They can be when in use with JOSE which we wouldn't be well aware of but I can make a note of the condition